### DISCUSSION TITLE
installation without docker.sock

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1286] [Ideas] [answered]
to increase security and run on Synology, TrueNAS, ... the container must run without docker.sock, as it is also handled by [iredmail all-in-one](https://hub.docker.com/r/iredmail/mariadb).

**Comment by Zoey2936** [comment, marked as answer]
See: https://github.com/nextcloud/all-in-one/tree/main/manual-install

**Comment by szaimen** [comment]
yes, manual-install as suggested by Zoey oder docker rootless when highest security is of concern: https://github.com/nextcloud/all-in-one/blob/main/docker-rootless.md
